### PR TITLE
adjust api docs

### PR DESCRIPTION
**Pull request by SantyWang**
Re: #

Changelog:
 *

### CODE REVIEW

File: cocos/core/director.ts [hunk 1]
```diff
@@ -172,29 +172,29 @@ export class Director extends EventTarget {
     public static readonly EVENT_BEFORE_COMMIT = 'director_before_commit';
 
     /**
-     * The event which will be triggered before the physics process.<br/>
-     * 物理过程之前所触发的事件。
+     * @en The event which will be triggered before the physics process.<br/>
```
Comment (pandamicro) [inline cocos/core/director.ts:175]: Remove `<br/>` check all existence in the file

File: cocos/core/geometry/curve.ts [hunk 2]
```diff
@@ -186,7 +186,8 @@ export class AnimationCurve {
     private cachedKey: OptimizedKey;
 
     /**
-     * 构造函数。
+     * @en Construct a curve with key frames
+     * @zh 通过关键帧构造一条曲线。
      * @param keyFrames 关键帧。
```
Comment (pandamicro) [inline cocos/core/geometry/curve.ts]: English doc

File: cocos/core/geometry/deprecated-3.0.0.ts [hunk 3]
```diff
@@ -151,8 +151,12 @@ function deprecatedClassMessage (oldClassName: string, newClassName) {
 }
 
 /**
+ * @en
  * Alias of [[Line]]
- * @deprecated Since v3.0
+ * @zh
+ * [[Line]] 别名类
+ *
+ * @deprecated Since v3.0, please use Line instead
```
Comment (pandamicro) [inline cocos/core/geometry/deprecated-3.0.0.ts:159]: Add `@internal` for all classes in this file

**Review by pandamicro — CHANGES_REQUESTED** [review]

File: cocos/core/geometry/deprecated-3.0.0.ts (hunk 3 above)
Comment (SantyWang) [reply cocos/core/geometry/deprecated-3.0.0.ts:159]: @internal will cause compiling failure

File: cocos/core/director.ts (hunk 1 above)
Comment (SantyWang) [reply cocos/core/director.ts:175]: seems will be invisible in final html

**Review by pandamicro — APPROVED** [review]

**Comment by pandamicro** [comment]
ignoring ESLinter issues which are not caused by the modification

Failed ones:

- ESLint check cocos/core/data/utils/attribute.ts
- ESLint check cocos/core/utils/id-generator.ts
- ESLint check cocos/core/utils/js-typed.ts
- ESLint check cocos/tween/actions/action-interval.ts

**Pull request merged**